### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by orangemug**
Changes

 - Update all the dependencies
 - Change from using `raw-loader` to `babel-plugin-static-fs` as it needs API updates anyway and seemed like a simpler approach

Build: https://1458-84182601-gh.circle-artifacts.com/0/artifacts/build/index.html

Does anybody fancy giving it a try, see if they can find anything broken?

### CODE REVIEW

**Comment by pathmapper** [comment]
Thanks @orangemug, I'll try this out.

**Comment by orangemug** [comment]
Thanks @pathmapper

File: package.json [hunk 1]
```diff
@@ -20,43 +20,43 @@
...
+    "file-saver": "^2.0.2",
     "jsonlint": "github:josdejong/jsonlint#85a19d7",
     "lodash.capitalize": "^4.2.1",
     "lodash.clamp": "^4.0.3",
     "lodash.clonedeep": "^4.5.0",
     "lodash.get": "^4.4.2",
     "lodash.isequal": "^4.5.0",
     "lodash.throttle": "^4.1.1",
-    "mapbox-gl": "^1.2.0",
+    "mapbox-gl": "^1.4.0",
```
Comment (pathmapper) [inline package.json]:
```suggestion
    "mapbox-gl": "^1.4.1",
```

v1.4.1 is released which fixes a warning for images with empty string value.

Comment (orangemug) [reply package.json]: Thanks @pathmapper change made

**Comment by pathmapper** [comment]
@orangemug this PR removes the `package-lock.json` and adds a `yarn.lock`. Was this intended?

**Comment by orangemug** [comment]
Yeah @pathmapper it was intentional, however now you've made me reconsider. I think yarn is 'better' because it appears to create a more consistent lockfile. However a lot of people won't have `yarn`installed. So I favour of making it easier to contribute to Maputnik, npm lockfiles are probably better.

I'll regenerate the `package-lock.json`and remove `yarn.lock`. Thanks for questioning it.

**Comment by orangemug** [comment]
> I'll regenerate the package-lock.json and remove yarn.lock. Thanks for questioning it.

I've updated the PR

**Comment by pathmapper** [comment]
@orangemug I didn't found anything broken and think it's good to merge. Also checked some changelogs if there were any breaking changes which could affect us.

**Comment by orangemug** [comment]
Thanks @pathmapper merging!

**Comment by pathmapper** [comment]
@orangemug what about a `v1.6.1` release including this PR and https://github.com/maputnik/editor/pull/555/commits/bda7a0e659cca4d4f863376c851fa23daac40d5c?

The not working OMT styles on https://maputnik.github.io/editor/ are bad UX ...

**Comment by orangemug** [comment]
> @orangemug what about a v1.6.1 release

@pathmapper I've released v1.6.1

 - https://github.com/maputnik/editor/releases/tag/v1.6.1
 - https://maputnik.github.io/editor

**Comment by pathmapper** [comment]
@orangemug great, I'll take care of the CLI binaries.

**Comment by orangemug** [comment]
> I'll take care of the CLI binaries.

Thanks

**Comment by pathmapper** [comment]
CLI binaries added: https://github.com/maputnik/editor/releases/tag/v1.6.1

**Pull request merged**